### ISSUE TITLE
[Adaptive Card iOS - TooltipTestCard.json]: Elements under 'TooltipTestCard,Json' are not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8399]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'TooltipTestCard.json' and activate it.
7. Perform the right swipe and observe the issue.

**Actual Result:**
While performing the right swipe, elements under 'Tooltiptestcard.Json' are not accessible.

**Expected Result:**
Elements under 'Tooltiptest.Json' screen should be accessible through the swipe gesture.

**Observation:**
This issue is working fine with Keyboard.

**User Experience:**
It will impact the screen reader user if any element on the screen is not accessible through the swipe gesture as user won't be able to perform any action.
 
**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html 

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227783611-2ab6ba8a-7c3e-4b45-a49b-f5d0bc942b41.MP4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
@vagpt 
Please note that any cards under the 'Test & Elements' section are intended solely for internal feature development. Once features have been verified, new cards are created and placed in the 'Scenario' section. It's important to focus testing efforts only on cards within the 'Scenario' section.

To address this, I'm currently working on fixing the issue so that cards from other folders will not appear in the sample app. Thank you for your understanding.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
